bson/src/main/org/bson/internal/PlatformUtil.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bson.internal;
+
+/**
+ * Utility class for platform-specific operations.
+ * This class is not part of the public API and may be removed or changed at any time.
+ */
+public final class PlatformUtil {
+
+    private PlatformUtil() {
+        //NOOP
```

Comment:
the above `//NOOP` is confusing. Usually such private method for util class is only meant for instance creation, and what to do in the constructor is not of interest.